sa/_db-next/migrations/20170718114448_DBCleanup.sql

```diff
@@ -0,0 +1,61 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+DROP TABLE externalCerts;
+
+DROP TABLE identifierData;
+
+ALTER TABLE certificateStatus DROP LockCol;
+
+ALTER TABLE certificateStatus DROP subscriberApproved;
+
+START TRANSACTION;
+ALTER TABLE certificateStatus ADD id bigint(20) NOT NULL;
```

Comment:
Nit: BIGINT should be capitalized. Also, how does this get initialized if it's required to be NOT NULL and isn't yet AUTO_INCREMENT? Have you tested this migration on a database that already has several entries in it?